sansa-query-spark/pom.xml

```diff
@@ -22,6 +22,13 @@
 			<artifactId>sansa-rdf-spark${scala.version.suffix}</artifactId>
 		</dependency>
 
+		<!-- SANSA RDF -->
```

Comment:
We are going to deprecate this reader on a new version of SANSA and for that there is no need to use this dependecny anymore!